src/vsr/superblock.zig

```diff
@@ -517,24 +515,21 @@ pub const ManifestReferences = struct {
     }
 };
 
-pub const FreeSetReference = struct {
-    /// Checksum over the entire encoded free set.
+pub const TrailerReference = struct {
+    /// Checksum over the entire encoded trailer.
     checksum: u128,
     last_block_address: u64,
     last_block_checksum: u128,
-    free_set_size: u32,
-    storage_size: u64,
+    trailer_size: u32,
```

Comment:
For generic trailer, let's future-proof this to u64?

Reply:
Should `free_set_size`/`client_sessions_size` in the `CheckpointState` be `u64` too?

Reply:
I’d say yes!